changelogs/fragments/alicloud_params_add.yml

```diff
@@ -0,0 +1,5 @@
+changes:
```

Comment:
You should mention that the modules now only support Python 3.6, and not older Python version. Also, you bumped the required libraries for some modules.